international401/Dependencies.toml

```diff
@@ -340,3 +338,12 @@ modules = [
 	{org = "ballerinax", packageName = "health.fhir.r4.international401", moduleName = "health.fhir.r4.international401"}
 ]
 
+[[package]]
+org = "ballerinax"
+name = "health.fhir.r4utils"
```

Comment:
Shall we check why the r4utils is needed here?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/efc9503f7d123ac8ce2bff328a5be6467fd22a3b